mat/hogsvd.go

```diff
@@ -158,30 +159,32 @@ func (gsvd *HOGSVD) Len() int {
 
 // UTo extracts the matrix U_n from the singular value decomposition, storing
 // the result in-place into dst. U_n is size r×c.
-// If dst is nil, a new matrix is allocated. The resulting U matrix is returned.
 //
-// UTo will panic if the receiver does not contain a successful factorization.
-func (gsvd *HOGSVD) UTo(dst *Dense, n int) *Dense {
+// If dst is empty, UTo will resize dst to be r×c. If dst is
+// non-empty, UTo will panic if dst is not r×c. UTo will also
```

Comment:
"When dst is non-empty, UTo will panic if dst is not r×c."

Reply:
Done.